exec/csnex/Exec.cs

```diff
@@ -26,6 +26,11 @@ public int Run(bool EnableAssertions)
         {
             ip = 0;
 
+            for (int g = 0; g < bytecode.globals.Capacity; g++)
+            {
+                bytecode.globals.Add(new Cell(Cell.Type.Address));
```

Comment:
No.  I looked at this closely, and was noticing that all my cells on the stack were ending up as `Type.Nothing`'s.  This didn't really affect execution, but it could at some point, so since all of the globals are forced to be used as Address variables initially, I just set them here.  Otherwise, I have no way of forcing a Cell to be Address type, without making a custom constructor, and force-passing the type.  Perhaps I should have a look at how this is handled in C++.

Reply:
Unlike the C++ code, I cannot pass a dereferenced cell into a Cell to create a new Address Cell.  If I do something like that, then I have to store my globals as a `List<Cell>`, rather than an array.  And since I'd be calling `new` each time it would make my `List<Cell>` management that much harder.  I felt this was the easier way.

Reply:
I do not recommend basing the C# implementation on the C++ code. The object semantics are too different. If you want to refer to an existing model, then look at the jnex code, it is much closer to C# semantics.

Reply:
Fixed with `Cell::set()` functions.